apps/desktop-wallet/src/components/DefaultAddressSwitch.tsx

```diff
@@ -73,7 +73,7 @@ const SelectCustomComponent = (value?: SelectOption<AddressHash>, disablePointer
       disablePointer={disablePointer}
       squared
     >
-      {value?.value && <AddressBadgeStyled addressHash={value.value} truncate />}
+      {value?.value && <AddressBadgeStyled addressHash={value.value} truncate disableCopy />}
```

Comment:
Easy fix, but that's actually one of the practical places where to copy the main address hash 🤔 
But let's do it that way for now.

Reply:
You can still copy it once you click on it (from the list, it will be the first item)